### ISSUE TITLE
change the screen refresh rate without touching the screen？

### ISSUE THREAD

**Issue by Yhgtw** [yc9559/dfps#27]
Thank you for allowing me to switch the screen refresh rate by modifying the dfps.txt file. However, I found that after modifying dfps.txt, I need to touch the screen for the refresh rate change to take effect. Is there a way to successfully change the screen refresh rate without touching the screen?

**Comment by valdy00** [comment]
I don't think so, cos this module works based on touch, means device go into idle state when no touch on the screen and vice versa.
